site/home.css

```diff
@@ -1,29 +1,63 @@
-
-html, body{
+html,
+body {
     height: 100%;
-    margin: 0px;
+    margin: 0;
 }
+
 .css-selector {
     height: 100%;
     background: linear-gradient(325deg, #08627d, #ce1395);
     background-size: 400% 400%;
     -webkit-animation: AnimationName 30s ease infinite;
     -moz-animation: AnimationName 30s ease infinite;
+    -o-animation: AnimationName 30s ease infinite;
     animation: AnimationName 30s ease infinite;
 }
 
 @-webkit-keyframes AnimationName {
-    0%{background-position:0% 50%}
-    50%{background-position:100% 51%}
-    100%{background-position:0% 50%}
+    0% {
```

Comment:
Lines 18-26 are repeated many times in this file, maybe the code could be refactored?

Reply:
Sure, @Inoxia25. Could probably do so using comma seperators. Also, maybe change AnimationName and .css-selector to something more semantic? 

Can open a new issue for this, possibly?

Reply:
@louisefindlay23 yes sounds great!